### PR TITLE
master: update release-tools

### PR DESCRIPTION
**Pull request by pohly**
Commit summary:
80678456 travis.yml: also use vendor directory
23df4aef prow.sh: use vendor directory if available
c8a1c4af better handling of Go version
5e773d2d update CI to use Go 1.13.3
e0fde8c4 Add new variables for 1.16 and remove 1.13
f1697d2c Do full git clones in travis. Shallow clones are causing test-subtree errors when the depth is exactly 50.
2d6b3ce8 Build Windows only for amd64
c1078a65 go-get-kubernetes.sh: automate Kubernetes dependency handling
194289aa update Go mod support
6208f6ab Enable hostpath expansion

```release-note
NONE
```

### CODE REVIEW

**Comment by xing-yang** [comment]
Let's get the split controller PR merged first.
/hold

**Comment by xing-yang** [comment]
The split controller PR is merged.  Can you rebase this PR?

**Comment by pohly** [comment]
Updated. Rebasing isn't possible when `csi-release-tools` is involved, so I added a merge commit where I resolved the conflict.

File: release-tools/prow.sh [hunk 1]
```diff
@@ -200,9 +206,10 @@ configvar CSI_PROW_HOSTPATH_CANARY "" "hostpath image"
 # all generated files are present.
 #
 # CSI_PROW_E2E_REPO=none disables E2E testing.
-configvar CSI_PROW_E2E_VERSION_1_13 v1.14.0 "E2E version for Kubernetes 1.13.x" # we can't use the one from 1.13.x because it didn't have --storage.testdriver
+# TOOO: remove versioned variables and make e2e version match k8s version
```
Comment (xing-yang) [inline release-tools/prow.sh]: s/TOOO/TODO?

Comment (pohly) [reply release-tools/prow.sh]: True, but that can't be fixed here. It went into csi-release-tools in https://github.com/kubernetes-csi/csi-release-tools/commit/e0fde8c4f5684f5c259c8c7a1d116def587cabf0 and would have to be fixed there.

Not worth it, right? :grimacing:

**Pull request merged**